thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/template.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if isSplit}}
   {{#each splitUrns as |urn|}}
-    <div>
-      <h4>{{get-safe splitLabels urn}}</h4>
+    <div class="row">
```

Comment:
fyi, no need for a row, if we're not going to use col-xs-... inside. 
This actually causes alignment issues since the col-xs-... classes offset the negative margins from `.row`s